meta-ov/recipes-core/psplash/files/psplash-rotation

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+
+# reads rotation from cmdline convert it to degrees and writes it to
+# /tmp/rotation for psplash
+
+rotation=/tmp/rotation
+
+read -r cmdline < /proc/cmdline
+for param in ${cmdline} ; do
+  case $param in
+      fbcon=*)
+          fbcon="${param#fbcon=}"
+          ;;
+  esac
+done
+
+case $fbcon in
+    "rotate:3")
+        echo "ROTATION=90" > "${rotation}"
+        ;;
+    "rotate:2")
+        echo "ROTATION=180" > "${rotation}"
+        ;;
+    "rotate:1")
+        echo "ROTATION=270" > "${rotation}"
+        ;;
+    *)
+        echo "ROTATION=0" > "${rotation}"
+        ;;
+esac
```

Comment:
I suspect (as @MaxKellermann suggests), that if you add this line here it will run just fine. Can you confirm / deny this?

```
exec /usr/bin/psplash --angle 90
```

Reply:
Yes it does. So the new version contains just that. I still find shell wrappers in systemd units just yuck.